sway-lib-std/src/option.sw

```diff
@@ -30,16 +30,14 @@ impl<T> Option<T> {
     /// Returns `true` if the result is [`Some`].
     fn is_some(self) -> bool {
         match self {
-            Option::Some(_) => true,
-            _ => false, 
+            Option::Some(_) => true, _ => false, 
```

Comment:
There are some formatting changes here that aren't directly related to this PR, can you remove them? Do them all in one go when resolving #2031